docs/developer/explanation/mail.rst

```diff
@@ -5,19 +5,13 @@ Launchpad Mail
 
 There are various kinds of emails in Launchpad:
 
-1. Mailing lists (represented by Launchpad teams). A mailing list has an
-   address ``TEAM_NAME@lists.launchpad.net``, archives
-   (``https://lists.launchpad.net/TEAM_NAME``), and an administrative
-   interface (``https://lists.canonical.com/mailman/admin/TEAM_NAME``).
-   Launchpad uses `Mailman <http://list.org>`__ to process these kinds
-   of mails.
-2. Emails sent from one user to another (that is, an email sent "by"
+1. Emails sent from one user to another (that is, an email sent "by"
    Launchpad, but really sent by user Alice when Alice uses the
    ``https://edge.launchpad.net/~barry/+contactuser`` form to contact
```

Comment:
Unrelated - but this URL cannot be correct. We only use "edge" for the new MP page.